app/views/find_local_council/_base_page.html.erb

```diff
@@ -4,13 +4,13 @@
         content="Find your local authority in England, Wales, Scotland and Northern Ireland" />
 <% end %>
 
-<main id="content" role="main">
+<main id="content" role="main" class="govuk-grid-column-two-thirds">
```

Comment:
Do these need to be [inside a `div.govuk-grid-row`](https://design-system.service.gov.uk/styles/layout/#common-layouts), or is that already catered for in the layout?

(It looks right from the screenshots, I just can't see where the row is coming from)

Reply:
I checked that too. It seems like it is (probably the grid-row class lives in the layout (or whatever outer view we have). Would be good to align the layout for all apps (potentially using https://govuk-frontend-review.herokuapp.com/full-page-examples/start-page as a reference).